Validation/SiTrackerPhase2V/python/Phase2ITValidateCluster_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from Validation.SiTrackerPhase2V.Phase2ITValidateCluster_cfi import Phase2ITValidateCluster
+clusterValidIT = Phase2ITValidateCluster.clone()
+
+from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
+premix_stage2.toModify(clusterValidIT,
+    pixelSimLinkSrc = "mixData:PixelDigiSimLink",
```

Comment:
should this rather be:
```suggestion
    InnerTrackerDigiSimLinkSource = "mixData:PixelDigiSimLink",
```

based on https://github.com/cms-sw/cmssw/pull/32151/files#diff-c321bd922c5db96fb61a4c6f6e3940a342eb4190cb9747377a65c89003564f6dR364
?